### DISCUSSION TITLE
The default network selected in the Torus Wallet shows 1 ETH = 1 USD

### DISCUSSION THREAD

**Discussion by pavelescurazvan** [Web3Auth/Web3Auth#962] [Help]
Hello,

We're integrating Web3Auth in our product, and as a first-time user, when you open the Torus Wallet, the "Ethereum Mainnet" is selected by default.

The problem is that the Torus Wallet shows 1 ETH to equal 1.00 USD.

Any idea what might be going on?

I am aware that I can go to Settings and refresh the network, which appears to fix the problem.

Tho - we can't ship this to our customers, with Torus Wallet defaulting to a bugged Ethereum network state, showing 1 ETH === 1 USD

Screenshot:
![Screenshot 2022-11-29 at 15 15 53](https://user-images.githubusercontent.com/25104492/204540849-d9ec2079-f7ff-43dc-aca3-b9bcf8b8c46c.png)

Thanks,
Razvan

**Comment by himanshuchawla009** [comment]
Thanks for reporting this issue, we are working on a fix. Will get back as soon as it is fixed and released.

**Comment by dzatorsky** [comment]
Any updated so far?

Also, it would be great to be able to choose the default network via SDK somehow. Couldn't find any options for `tourus.showWallet("home")`
